distribution/changelog.txt

```diff
@@ -28,6 +28,7 @@
 - Change: [#21214] Wacky Worlds and Time Twister’s scenario names now match their park names.
 - Change: [#21991] UI themes JSON now use colour names and a translucency bool, instead of a number (old themes still work).
 - Change: [#22057] Reorder Time Twister’s scenarios and adjust their difficulty classification.
+- Change: [#22196] Make track nagivation buttons holdable.
```

Comment:
There's a typo in the word navigation.

Reply:
Oops, my bad. I'll open a PR to amend it.